lib/smack/IntegerOverflowChecker.cpp

```diff
@@ -119,6 +119,21 @@ bool IntegerOverflowChecker::runOnModule(Module& m) {
     if (Naming::isSmackName(F.getName()))
       continue;
     for (inst_iterator I = inst_begin(F), E = inst_end(F); I != E; ++I) {
+      // Add check for UBSan left shift
+      if (auto chkshft = dyn_cast<CallInst>(&*I)) {
+        if (chkshft->getCalledFunction() != nullptr &&
```

Comment:
Maybe introduce a local function variable here so that you avoid calling `getCalledFunction` so many times.
Everything else looks good. So please squash these commits and in your final commit messages say that it `Fixes #383`.
Thanks!

Reply:
Done. Poor Travis.